### DISCUSSION TITLE
Exception Occured: org.hibernate.exception.JDBCConnectionException: Could not extract ResultSet

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#384] [General]
Dear All. I am getting above error which explains that my DB connection is not getting ready or there is a time out in creating the connection so I just want to know how can I solve this keeping in mind that I am using docker container with Keyfactor/ejbca-ce 8.0 and a separate mysql server 8.0.

**NOTE:** my mysql server is running fine with no delays and my mysqlclient from the front end server (hosting adminweb+RA web) is also creating the connection and running fine with any delays.

Thanks

**Comment by primetomas** [comment]
You need to provide the full stack trace to see what it originates from.
